### DISCUSSION TITLE
Ignore new indexes on init, is it possible?

### DISCUSSION THREAD

**Discussion by Asafb26** [BeanieODM/beanie#799] [Question] [answered]
I have a case which require me to connect to a remote db from local environment, for this case I have a read only user configured in atlas.
the current behavior of beanie AFAIK is to create new indexes in found in my codebase, but I want to ignore it for this case, is is possible?

**Comment by roman-right** [comment, marked as answer]
Hi @Asafb26 ,
We had a discussion about a similar problem in Discord. I plan to add this option to the init_beanie function this weekend.
